python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -262,6 +264,37 @@ def init_flashinfer_handlers(
         )
 
 
+@triton.jit
+def create_kv_indices_trition(
```

Comment:
triton?

Reply:
Yes. The torch.cat([... for i in range(batch)]) is too slow when large batch, so a simple triton kernel is used here.

Reply:
fixed.